graphene_django/converter.py

```diff
@@ -297,10 +281,16 @@ def dynamic_type():
         if not _type:
             return
 
-        return Field(
+        if isinstance(field, models.OneToOneRel):
+            description = get_django_field_description(field.field)
+        else:
+            description = get_django_field_description(field)
+
+        return DjangoInstanceField(
```

Comment:
Shouldn't we also lookup the primary field of the model and pass the result in as `unique_fields`? I could see a model having a different primary key field name and I think that would cause this to break